### ISSUE TITLE
Inconsistent behavior when `source .venv/Scripts/activate` on Windows

### ISSUE THREAD

**Issue by jfcherng** [astral-sh/uv#1518]
When in git-bash (https://gitforwindows.org/) shell on Windows,

## With `pip`

```bash
python -m venv .venv
source .venv/Scripts/activate
echo $PATH  # ".venv/Scripts" is added into PATH
```

## With `uv`

```bash
uv v
source .venv/Scripts/activate
echo $PATH  # ".venv/bin" is added into PATH, and this is wrong
```

**Comment by jfcherng** [comment]
I can confirmed my issue is resolved in v0.1.4. Thanks!

**Comment by charliermarsh** [comment]
Awesome, thank you for following up!
